### DISCUSSION TITLE
隐藏后台无效,后台不强,跳过失效

### DISCUSSION THREAD

**Discussion by lingzeyang** [gkd-kit/gkd#85] [Q&A]
1. 应用内打开后台隐藏后在最近任务里仍然有gkd进程
2. 在给app加锁 ,禁止电池优化, 打开完全后台 app仍然出现被杀情况, 老版本更严重,后续版本被杀的情况减少了,但仍有
3. 打开app确认各服务正常,但打开开屏广告app无法触发跳过, 该app我确定在规则库里有.

**Comment by lisonge** [comment]
> 应用内打开后台隐藏后在最近任务里仍然有gkd进程

https://github.com/gkd-kit/gkd/issues/56

---

> 打开app确认各服务正常,但打开开屏广告app无法触发跳过, 该app我确定在规则库里有

https://github.com/gkd-kit/gkd/issues/64

**Comment by 888818** [comment]
安卓版快捷指令app 需要添加去广告规则

**Comment by DDPwang** [comment]
后台留存能力还不够，用着用着就失效了，需要重新打开软件，后台进程可以看到GKD，但是功能失效，自启动开启，后台上锁，
